App.js

```diff
@@ -120,9 +126,42 @@ export default class App extends React.Component {
           <StatusBar style="hidden" />
         </View>
       );
+    } else if (!image && !camera && textAsImage) {
+      if (text === null) {
+        return (
+          <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center', backgroundColor: 'black' }}>
+            <TextInputBox text={''} onSubmitPress={this._setText}/>
+            <FontDropDown textFont={textFont} onSelect={this._setFont}/>
+            <FloatingToolbar top={true} left={true}>
+              <ActionButton onPress={this._closeTextAsImage} text={i18n.t("button_back")} textPosition="right" iconName="md-arrow-back" />
+            </FloatingToolbar>
+          </View>
+        );
+      }
+      else {
+        return (
+          <View style={{ flex: 1, backgroundColor: 'black' }}>
```

Comment:
This screen definition seems to be copy-pasted from the image-based drawing screen, and looks quite similar. Would it be possible to reuse reuse the original screen definition for both cases, but different parameterization?

Reply:
I made changes as you said, its now using the same screen definition for image and text as image.
Combined _resetImage and _resetText as I thought it would be cleaner

Reply:
Looks good, thanks!